docs/core/setup/how-to-detect-installed-versions.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Check which .NET Core versions on Windows, Linux, and macOS
+description: Learn how to list which versions of .NET Core are installed on your computer. This includes the .NET Core runtime and SDK.
+author: thraka
+ms.author: adegeo
+ms.date: 11/01/2019
+zone_pivot_groups: operating-systems-set-one
+---
+
+# How to check if .NET Core is already installed
+
```

Comment:
Should you make clear that this article requires that you have the SDK installed already? And how would you check which versions of the runtime you have installed without having the SDK installed?

Reply:
you don't need the SDK to use these commands.

Reply:
I didn't know you could do this just with the runtime!

So should you mention that if the commands fail to work it means you don't have any version installed?

Reply:
Good idea!

Reply:
It also requires 2.0+ since those options AFAIK were only introduced in 2.0.

Reply:
2.0 is the lowest supported .NET Core version.